docs/api-guide.md

```diff
@@ -645,11 +676,71 @@ List<Result> results = transaction.scan(scan);
 {% capture notice--info %}
 **Note**
 
-You can't specify clustering-key boundaries and orderings in `Scan` without specifying a partition key.
+You can't specify any filtering conditions and orderings in cross-partition `Scan` except for JDBC databases. See the following section to use cross-partition `Scan` with filtering or ordering for JDBC databases.
+{% endcapture %}
+
+<div class="notice--info">{{ notice--info | markdownify }}</div>
+
+##### Execute cross-partition `Scan` with filtering and ordering
+
+By enabling the cross-partition scan option with filtering and ordering for JDBC databases as follows, you can execute a cross-partition `Scan` operation with flexible conditions and orderings.
+
+```properties
+scalar.db.cross_partition_scan.enabled=true
+scalar.db.cross_partition_scan.filtering.enabled=true
+scalar.db.cross_partition_scan.ordering.enabled=true
+```
```

Comment:
It seems that the description of this parameter is added to ScalarDB SQL document in PR https://github.com/scalar-labs/scalardb-sql/pull/347.

However, I think it might be better to add descriptions in the `docs/configurations.md` on the ScalarDB (`scalardb` repository) side since we can use related API in the ScalarDB core API.

What do you think?

Reply:
Definitely, I will revise both.

Reply:
Fixed in 6726530.